cli/command/swarm/opts.go

```diff
@@ -24,13 +24,17 @@ const (
 	flagToken               = "token"
 	flagTaskHistoryLimit    = "task-history-limit"
 	flagExternalCA          = "external-ca"
+	flagKeepOldSnapshots    = "keep-old-snapshots"
```

Comment:
Not entirely sure about this flag's name. Name feels like a "boolean" "Yes, keep old snapshots"

It's a bit like log-rotation (https://docs.docker.com/engine/admin/logging/overview/#/json-file-options), `max-snapshots`?

Is there a reason we're specifying _old_ snapshots? Would it make sense to specify the _total_ (including current)?